test/controllers/api/players_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class Api::PlayersControllerTest < ActionDispatch::IntegrationTest
```

Comment:
If your tests files does nothing is better don't commit them yet.

Reply:
Those test are autogenerated by rails, do you suggest to delete it?

Reply:
No, don't add this file to the `.gitignore`, if you do that then you have to create a commit ti make the changes be reflected and then when you start using it you'll have to remove it from the gitignore again, creating another commit. Also... if this was autogenerated you can leave the file as it is, there's no problem on pushing it (cc @diegosaul402)